### PR TITLE
Address comments during the WG adoption call

### PR DESCRIPTION
**Pull request by tireddy2**

### CODE REVIEW

File: draft-ietf-jose-pqc-kem-00.md [hunk 1]
```diff
@@ -0,0 +1,506 @@
+---
+title: "Post-Quantum Key Encapsulation Mechanisms (PQ KEMs) for JOSE and COSE"
+abbrev: "PQ KEM for JOSE and COSE"
+category: std
+
+docname: draft-reddy-cose-jose-pqc-kem
```
Comment (auriee) [inline draft-ietf-jose-pqc-kem-00.md:6]: draft-ietf-jose-pqc-kem

File: draft-ietf-jose-pqc-kem-00.md [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+     
+--- abstract
+
+This document describes the conventions for using Post-Quantum Key Encapsulation Mechanisms (PQ-KEMs) within JOSE and COSE. 
+
+--- middle
+
+# Introduction
+
+Quantum computing is no longer perceived as a conjecture of computational sciences and theoretical physics.  Considerable research efforts and enormous corporate and government funding for the development of practical quantum computing systems are being invested currently. As such, as quantum technology advances, there is the potential for future quantum computers to have a significant impact on current cryptographic systems.
```
Comment (auriee) [inline draft-ietf-jose-pqc-kem-00.md:97]: replace "conjecture" with "consequence"

File: draft-ietf-jose-pqc-kem-00.md (hunk 1 above)
Comment (tireddy2) [reply draft-ietf-jose-pqc-kem-00.md]: Fixed

File: draft-ietf-jose-pqc-kem-00.md (hunk 2 above)
Comment (tireddy2) [reply draft-ietf-jose-pqc-kem-00.md]: Done

**Review by auriee — APPROVED** [review]

**Pull request merged**